src/demo/java/com/amplitude/DemoController.java

```diff
@@ -10,7 +10,7 @@
 	public String index() {
 		Amplitude amplitude = Amplitude.getInstance("INSTANCE_NAME");
 		amplitude.init("8e07b9d451a7d07bd33f6e9ba5870f21");
-		amplitude.logEvent("Test Event");
+		amplitude.logEvent("Event name test Apr 14th 3:40 PM");
```

Comment:
Minor thing but I think we should have a better name for the test logEvent.

Reply:
+1

Reply:
Yeah, oops, I'll fix it